Sources/URL++.swift

```diff
@@ -46,5 +46,12 @@ extension URL {
         guard let scheme = self.scheme, let otherScheme = otherUrl.scheme else { return false }
         return scheme.caseInsensitiveCompare(otherScheme) == .orderedSame
     }
-
+  
+    var queryParamsForSSO: [String : String] {
+        guard let host = self.host else { return [:] }
+        return host.split(separator: "&").reduce(into: [String:String](), {
+            let keyValue = $1.split(separator: "=")
+            $0[String(keyValue[0])] = String(keyValue[1])
```

Comment:
nit: I'd prefer if the arguments were named

Reply:
Fixed!
https://github.com/mattdonnelly/Swifter/pull/271/commits/3987aa1172d3bde95d6a4ceeac4615c692cc27b3